cloud/aws/bin/pgadmin_test.py

```diff
@@ -0,0 +1,113 @@
+import unittest
+
+from cloud.aws.bin.pgadmin import CIDRInputStateMachine
+"""
+ Tests for the CIDRInputStateMachine.
+
+ To run the tests: PYTHONPATH="${PYTHONPATH}:${pwd}" python3 cloud/aws/bin/pgadmin_test.py
+"""
+
+
+def ip(ip):
+    return lambda: ip
+
+
+class TestCIDRInputStateMachine(unittest.TestCase):
```

Comment:
Do we have github action that runs unit tests for this repo?

Reply:
Yes, we have the `PyTest / pytest (pull_request)` check which shows running the test file in its output:

```
cloud/aws/bin/pgadmin_test.py ........                                   [ 25%]
```